v2/split_slice.go

```diff
@@ -0,0 +1,29 @@
+package pie
+
+// SplitSlice returns splited slices which length equals splitLength.
+//
+// for this [1,2,3] slice with splitLength == 2 will return [][]int{{1,2}, {3}}.
+// If input slice length is less than splitLength, for this [1,2,3] slice with splitLength == 4
+// 		will return [][]int{{1,2,3}}.
+// If input slice length == 0 or splitLength <= 0, will return [][]T{}.
+func SplitSlice[T any](ss []T, splitLength int) [][]T {
```

Comment:
`SplitSlice` seems misleading to me because split is so overloaded in languages to mean that it's using another value as a separator. How about `Chunk`?

Reply:
you are right.